.dockerignore

```diff
@@ -3,3 +3,4 @@ README.md
 .git
 .tox
 dockerfiles
+.docker
```

Comment:
should this be added to `.gitignore` too? Usually `.dockerignore` should include all `.gitignore` content plus `.git`, when `COPY .` used in dockerfile.